src/services_models/reserves.jl

```diff
@@ -44,13 +54,12 @@ function service_requirement_constraint!(
         container = get_parameter(
             container,
             RequirementTimeSeriesParameter(PSY.Deterministic, "requirement"),
```

Comment:
Parameters no longer use this syntax. Are you sure this function is being tested?